### ISSUE TITLE
Add the Fuse Integration Services 2.0 templates to the CDK

### ISSUE THREAD

**Issue by chirino** [projectatomic/adb-utils#209]
The 2.0 version of Fuse Integration Service is about to release and it would be nice if it's OCP templates get included in the CDK.  The latest ER of the templates can be viewed at:

https://github.com/jboss-fuse/application-templates/tree/application-templates-2.0.redhat-000027

**Comment by chirino** [comment]
Do you guys need the images to exist before pulling in the templates that reference them?  The images wont be publicly available for like another month.

**Comment by LalatenduMohanty** [comment]
@chirino Yes, we need the availability of the images in image stream. But without the images how are you testing the templates?

**Comment by chirino** [comment]
We have the images in private (VPN accessible) repositories.

**Comment by chirino** [comment]
Any updates?

**Comment by LalatenduMohanty** [comment]
@chirino I have added a comment in your PR i.e. https://github.com/projectatomic/adb-utils/pull/216#issuecomment-285363411  and status is currently the same.

**Comment by LalatenduMohanty** [comment]
@chirino Minishift has t[he addon feature](https://github.com/minishift/minishift/blob/master/docs/source/using/managing-minishift.adoc#addons)  now which will allow us to add extra templates to CDK and make it default
